recipes/opencascade/all/conanfile.py

```diff
@@ -412,8 +412,8 @@ def _patch_sources(self):
 
         # Honor fPIC option, compiler.cppstd and compiler.libcxx
         replace_in_file(self, occt_defs_flags_cmake, "-fPIC", "")
-        replace_in_file(self, occt_defs_flags_cmake, "-std=c++0x", "")
-        replace_in_file(self, occt_defs_flags_cmake, "-std=gnu++0x", "")
+        replace_in_file(self, occt_defs_flags_cmake, "-std=c++0x", "", strict=False)
+        replace_in_file(self, occt_defs_flags_cmake, "-std=gnu++0x", "", strict=False)
```

Comment:
Please no strict=False, keep this under control so that we can know for which version it's really required.

Reply:
You have to properly take into account modifications of https://github.com/Open-Cascade-SAS/OCCT/commit/e92d322d4090d9611930bf7cabcc74fa981432b0, in order to honor compiler.cppstd from profile.